schoolopy/main.py

```diff
@@ -317,7 +312,7 @@ def get_section(self, section_id):
         :param section_id: ID of section on which to get data.
         :return: Section object.
         """
-        return Section(self._get('sections/%s' % section_id))
+        return Section(self._get(f"sections/{section_id}"))
```

Comment:
I would prefer to use single quotes here too, and to not use `f''` strings because we don't use them elsewhere and unless we're going to switch the whole codebase to use them (because there's lots of opportunities to) I don't think it's worth breaking compatibility with python <3.8

Reply:
Can you remove the `f''` string here?